client/my-sites/plugins-wpcom/default-plugins.js

```diff
@@ -165,9 +165,9 @@ export const defaultPremiumPlugins = [
 export const defaultBusinessPlugins = [
 	{
 		name: i18n.translate( 'Google Analytics' ),
-		descriptionLink: '/plans/features/google-analytics/{siteSlug}',
+		descriptionLink: '/categorys/features/google-analytics/{siteSlug}',
```

Comment:
question marks?

not sure I understand what you are commenting here @gwwar :wink:

maybe this is a typo and it needs to remain `plans` - I did a find/replace so that's probably it